backend/src/atlas_assistant/tools/sql.py

```diff
@@ -184,6 +186,11 @@ def get_prompt(dataset: Dataset) -> str:
 
 {dataset.get_head_table()}
 
+Other instructions:
+
+    - Whenever summing over numeric values, you must include a `where` clause
+      that removes all `nan` values using the DuckDB `isnan` function
```

Comment:
Ahh, interesting. I added some bits about adding `CASE` statements when aggregating data... maybe we can chat about the options tomorrow.